rpms/python-amqp/python-amqp.spec

```diff
@@ -0,0 +1,133 @@
+%global srcname amqp
+
+
+Name:           python-%{srcname}
+Version:        1.4.9
+Release:        2%{?dist}
+
+License:        LGPLv2+
+Summary:        Low-level AMQP client for Python (fork of amqplib)
+Group:          Development/Languages
+URL:            http://pypi.python.org/pypi/amqp
```

Comment:
It'd be nice if this URL and the Source0 URL used HTTPS explicitly. Not a huge deal, as ``pypi.python.org`` redirects to HTTPS.